### DISCUSSION TITLE
Button Grid - 3D printing

### DISCUSSION THREAD

**Discussion by tnph** [OMOTE-Community/OMOTE-Firmware#35] [Q&A]
I'm having difficulty printing the button grid - specifically the centre button in the middle of the 4-way arrows becomes attached to outer ring and won't budge, the tolerances are very small!  I've tried printing it with the Cura "Extra fine 0.06mm" profile and others on an Ultimaker S5 but only managed one successfull print. 
Questions:
1. any tips on the best settings in Cura
2. is it possible to get hold of CAD files compatible with FreeCAD so I can redesign this part.  I had the idea of removing the centre button from the button grid and creating a top hat style centre button (like the colour buttons) with appropriate mods to the 4-way arrow button.

**Comment by Dark1886** [comment]
I experienced a similar problem before I tuned my printer a little better. It's possible your printer is extruding a little to much which causes it to push outside the intended boundaries. 

-Possible fixes-
1. You can attempt to do some flow calibration if you have not done that before. 
2. Horizontal compensation (XY compensation on Prusaslicer) and use a small negative value (starting at -0.1 to -0.02 and increase it needed). This will help expand the space between the parts, although it will also mess with the images on the buttons a little if you go too high.

**Reply by tnph** [reply]
Thanks Dark1886, I'll give those options a go.  I use Cura for slicing and there are a million options in "expert" mode!!

**Reply by Dark1886** [reply]
No problem, let me know how it turned out. 

If you haven't gone through full 3d printer tuning or begin seeing certain problems, have a look at this guide. It may mention voron/klipper in it, but the guide still stands well for all slicers/printers. 

My current recommendations if you see problems with smaller gaps welding together
1. XY compensation/Horizontal expansion - This is kind of a 'band-aid' fix since it doesn't truly solve the root cause, but it is easy, and sometimes necessary. but it's better to not depend on it if possible.
2. If the first layer/few layers are all that is fusing, use elephant foot compensation/first layer horizontal expansion. Its the same setting as above, but only for the first layer/few layers. 
3. possible slow your printer down if it's going to quick, maybe just the acceleration value. 
4. Tune the flow of your printer. If you're extruding a little too much plastic, it will start 'leaking' into areas you didn't intend.
